api/server-server/definitions/event-schemas/m.signing_key_update.yaml

```
@@ -0,0 +1,68 @@
+# Copyright 2020 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+type: object
+title: m.signing_key_update
+description: |-
+  An EDU that lets servers push details to each other when one of their users
+  updates their cross-signing keys.
+allOf:
+  - $ref: ../edu.yaml
+  - type: object
+    properties:
+      edu_type:
+        type: enum
+        enum: ['m.signing_key_update']
+        description: The string ``m.signing_update``.
+        example: "m.signing_key_update"
+      content:
+        type: object
+        description: The updated signing keys.
+        title: Signing Key Update
+        properties:
+          user_id:
+            type: string
+            description: The user ID whose cross-signing keys have changed.
+            example: "@alice:example.com"
+          master_key:
+            type: object
+            allOf:
+              - $ref: ../../../client-server/definitions/cross_signing_key.yaml
+              # FIXME: why isn't the doc generator picking up this example?
+              - example: {
```

have you tried putting the `example` directly under `master_key` rather than an entry in the `allOf` list?
```suggestion
            example: {
```